packages/edit-site/src/style.scss

```diff
@@ -0,0 +1,58 @@
+@import "./components/block-editor/style.scss";
+@import "./components/header/style.scss";
+@import "./components/notices/style.scss";
+@import "./components/sidebar/style.scss";
+
+// In order to use mix-blend-mode, this element needs to have an explicitly set background-color.
+// We scope it to .wp-toolbar to be wp-admin only, to prevent bleed into other implementations.
+html.wp-toolbar {
+	background: $white;
+}
+
+body.gutenberg_page_gutenberg-edit-site {
+	@include wp-admin-reset(".edit-site");
+}
+
+.edit-site,
+// The modals are shown outside the .edit-site wrapper, they need these styles.
+.components-modal__frame {
+	@include reset;
+}
+
+.edit-site {
+	// On mobile the main content area has to scroll, otherwise you can invoke
+	// the over-scroll bounce on the non-scrolling container, for a bad experience.
+	@include break-small {
+		bottom: 0;
+		left: 0;
+		min-height: calc(100vh - #{$admin-bar-height-big});
+		position: absolute;
+		right: 0;
+		top: 0;
+	}
+
+	// The WP header height changes at this breakpoint.
+	@include break-medium {
+		min-height: calc(100vh - #{$admin-bar-height});
+	}
+
+	> .components-navigate-regions {
+		height: 100%;
+	}
+}
+
+/**
+ * Animations
+ */
+
+// These keyframes should not be part of the _animations.scss mixins file.
+// Because keyframe animations can't be defined as mixins properly, they are duplicated.
+// Since they are intended only for the editor, we add them here instead.
+@keyframes edit-post__fade-in-animation {
+	from {
+		opacity: 0;
+	}
+	to {
+		opacity: 1;
+	}
+}
```

Comment:
One of the reasons of the introduction of the EditorRegions component was to abstract the layout of the editor page to use it elsewhere instead of duplicating all these stylesheets. It could be an opportunity to use it.

Reply:
Yes, I thought/tried that. The problem is that it's still very specific to `edit-post`, we would have to move it to `block-editor` and make it a bit more abstract maybe?

I think that it won't take long before the `edit-post` and `edit-site` layouts diverge so much that it will become impractical to share something like `EditorRegions`.

I do see potential for something similar to `EditorRegions`, but specific to `edit-site`, later down the road when the desired layout is clearer.

Reply:
What's specific to `edit-post` in that package is just  the aria regions labels. Other than that I think it's generic enough. Tbh,  I feel we should do the refactoring now rather than after the fact.

Reply:
Move `EditorRegions` to `editor` and use it here? I don't think that will be harder to do later, but we could do it.

Reply:
I'm not sure yet which package, could be `block-editor` maybe? I don't know yet since it's basically just a UI component but maybe not generic enough for `@wordpress/components`?

Reply:
Seems more like an `editor` thing since it deals with the area that surrounds the `block-editor`.